fission-cli/library/Fission/CLI/Internal/ServerDID.hs

```diff
@@ -0,0 +1,11 @@
+{-# OPTIONS_GHC -fno-warn-incomplete-uni-patterns #-}
```

Comment:
i.e. we know that we don't handle the case where we can't decode the DID. This specific case will succeed, so disable the check.